compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -13570,10 +13725,18 @@ TR::Node *ifCmpWithEqualitySimplifier(TR::Node * node, TR::Block * block, TR::Si
 
    makeConstantTheRightChildAndSetOpcode(node, firstChild, secondChild, s);
 
+   TR::ILOpCodes opCode = node->getOpCodeValue();
```

Comment:
not a big deal, but why did this definition move here?

Reply:
Historical reasons :sweat_smile: Fixed.